docker-compose.yml

```diff
@@ -29,8 +29,15 @@ services:
       - CHROMA_OTEL_SERVICE_NAME=${CHROMA_OTEL_SERVICE_NAME}
       - CHROMA_OTEL_GRANULARITY=${CHROMA_OTEL_GRANULARITY}
       - CHROMA_SERVER_NOFILE=${CHROMA_SERVER_NOFILE}
+    restart: unless-stopped # possible values are: "no", always", "on-failure", "unless-stopped"
     ports:
-      - 8000:8000
+      - "8000:8000"
+    healthcheck:
+      # below test does not require curl or wget to be installed
+      test: [ "CMD", "/bin/bash", "-c", "cat < /dev/null > /dev/tcp/localhost/8000" ]
```

Comment:
Do we have a health endpoint? if not, I am ok with this for now.

Reply:
is wget not already in the image? I think it would be nice to use the heartbeat endpoint. Installing curl/wget is ok I think no?

Reply:
let's add curl then